packages/components/addon/components/hds/alert/index.js

```diff
@@ -1,6 +1,24 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
+export const DEFAULT_TYPE = 'page';
+export const TYPES = ['page', 'inline', 'compact', 'toast'];
+export const DEFAULT_COLOR = 'neutral';
+export const COLORS = [
+  'critical',
+  'warning',
+  'neutral',
+  'highlight',
+  'success',
+];
+export const MAPPING_COLORS_TO_ICONS = {
```

Comment:
@amyrlam notice: for now this works perfectly, but I've just realized that these icons can be filled or not depending if they are in the "compact" alert or not, so it's likely we'll have to move this mapping logic inside the `get icon()` method directly, and use the `@type/this.type` as an extra condition to determine which icon to use 

@amyrlam @Dhaulagiri @MelSumner can you see now the power of moving the logic in the "right" place? can you imagine doing all these conditionals only at template level?

Reply:
We talked about ^^ in a meeting, I'll address the compact type shortly. I get what you're saying now. Compact type has a different mapping of which default icon it informs. It will have some .hbs logic as well, as certain elements will be hidden, but can discuss when up soon